david-vazquez/challenges/react-tour-of-heroes/my-app/src/components/list/List.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function List() {
+  const heroes = [
+    { id: 11, name: 'Dr Nice' },
+    { id: 12, name: 'Narco' },
+    { id: 13, name: 'Bombasto' },
+    { id: 14, name: 'Celeritas' },
+    { id: 15, name: 'Magneta' },
+    { id: 16, name: 'RubberMan' },
+    { id: 17, name: 'Dynama' },
+    { id: 18, name: 'Dr IQ' },
+    { id: 19, name: 'Magma' },
+    { id: 20, name: 'Tornado' },
```

Comment:
You should save this constant in another file, and import it when you need ir. DRY